src/app/services/wallet.service.ts

```diff
@@ -64,16 +65,31 @@ export class WalletService {
       const minRequiredOutputs =  this.getMinRequiredOutputs(amount, outputs);
       const totalCoins = parseInt((minRequiredOutputs.reduce((count, output) =>
         count + output.coins, 0) * 1000000) + '', 10);
-      const totalHours = minRequiredOutputs.reduce((count, output) => count + output.hours, 0);
+
+      if (totalCoins < amount * 1000000) {
```

Comment:
The result of the multiplication must be filtered with `parseInt`, to prevent errors.